src/realm/object-store/results.hpp

```diff
@@ -374,6 +405,9 @@ class Results {
     auto dispatch(Fn&&) const REQUIRES(!m_mutex);
 
     enum class EvaluateMode { Count, Snapshot, Normal };
+    /// Return true if the Results has changed since the last call to
+    /// `has_changed()`. Note that this function is idempotent.
```

Comment:
How can this function be idempotent? If calling it twice returns the same value both times then it's not returning true only if the Results has changed since the last call.

Reply:
good catch, I'll reword the doc